src/workers/extractCVMarkdown.ts

```diff
@@ -0,0 +1,47 @@
+import { CandidatePreferenceUpdated } from '@dailydotdev/schema';
+import type { TypedWorker } from './worker';
+import { UserCandidatePreference } from '../entity/user/UserCandidatePreference';
+import { extractMarkdownFromCV } from '../common/brokkr';
+
+export const extractCVMarkdown: TypedWorker<'api.v1.candidate-preference-updated'> =
+  {
+    subscription: 'api.extract-cv-markdown',
+    parseMessage: ({ data }) => CandidatePreferenceUpdated.fromBinary(data),
+    handler: async ({ data }, con, logger) => {
+      const { userId, cv, cvParsedMarkdown } = data.payload || {};
+      const blobName = cv?.blob;
+      const bucketName = cv?.bucket;
+      if (!!cvParsedMarkdown) {
```

Comment:
Edge-case but what about when user re-uploads?
Maybe we can simply clear cvParsedMarkdown on the deletion of cv?

Reply:
Already added that in src/schema/users.ts :)

Reply:
Ah yeah see it now, I hate this minified review mode 😅